unit_testing/cli/test_cli_sct_label_vertebrae.py

```diff
@@ -1,20 +1,37 @@
-from pytest_console_scripts import script_runner
-import pytest
-import logging
-import spinalcordtoolbox.scripts.sct_label_vertebrae as sct_label_vertebrae
-logger = logging.getLogger(__name__)
 import os
-import nibabel as nib
+import logging
+
+import pytest
+
 from spinalcordtoolbox.image import Image, compute_dice
 from spinalcordtoolbox.labels import check_missing_label
+import spinalcordtoolbox.scripts.sct_label_vertebrae as sct_label_vertebrae
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.sct_testing
+def test_sct_label_vertebrae_consistent_disc(tmp_path):
+    """Check that all expected output labeled discs exist"""
+    command = '-i sct_testing_data/t2/t2.nii.gz -s sct_testing_data/t2/t2_seg-manual.nii.gz -c t2 -discfile sct_testing_data/t2/labels.nii.gz -ofolder ' + str(tmp_path)
+    sct_label_vertebrae.main(command.split())
```

Comment:
Ahh, I see what went wrong here. This test had already been written prior to this PR -- I just slapped a `@pytest.mark.sct_testing` on it, rather than checking the test itself.

Thanks for calling me out on this. :smile:

Reply:
1f19ad2